nutils/function.py

```diff
@@ -1109,6 +1025,15 @@ def _join_arguments(args_list: Iterable[Mapping[str, Argument]]) -> Dict[str, Ar
 
 # CONSTRUCTORS
 
+HANDLED_FUNCTIONS = {}
+
+def implements(np_function):
+  'Register an __array_function__ or __array_ufunc__ implementation for Array objects.'
```

Comment:
Suggestion: use double bacticks for `__array_function__` and `__array_ufunc__`.

Reply:
done